release_notes/ose_3_1_release_notes.adoc

```diff
@@ -42,6 +42,13 @@ including any additional steps that may be required for a specific release.
 ====
 
 .For Administrators:
+* Several services and packages have been renamed:
```

Comment:
This mentions only "services and packages", but the list also includes renamed directory and file names.